examples/gnrc_networking_mac/Makefile

```diff
@@ -33,8 +33,10 @@ USEMODULE += ps
 USEMODULE += netstats_l2
 USEMODULE += netstats_ipv6
 USEMODULE += netstats_rpl
-# Use LWMAC as the MAC layer protocol
-USEMODULE += gnrc_lwmac
+# Use GoMacH as the MAC layer protocol
+USEMODULE += gnrc_gomach
+# Uncomment the following line to use LWMAC as the MAC layer protocol
```

Comment:
I guess you also have to put the line above into comments as well? Please specify that.

Reply:
You mean put `USEMODULE += gnrc_gomach` also into comment?

Reply:
Addressed.

Reply:
No. Just the statement to the comment that one needs to put `USEMODULE += gnrc_gomach` into a comment and uncomment `USEMODULE += gnrc_lwmac` if they want to use LWMAC instead of GoMacH (AFAIK you can't use both at the same time at the moment, right?)

Reply:
Yap, addressed again~